geo/src/test/java/org/odk/collect/geo/GeoUtilsTest.java

```diff
@@ -52,12 +59,27 @@ public void test_formatLocationResultString() {
     @Test
     public void capitalizesGps() {
         String input = "gps";
-        assertEquals("GPS", GeoUtils.capitalizeGps(input));
+        assertEquals("GPS", capitalizeGps(input));
 
         String locationProvider = "network";
-        assertEquals("network", GeoUtils.capitalizeGps(locationProvider));
+        assertEquals("network", capitalizeGps(locationProvider));
 
         String nullLocationProvider = null;
-        assertNull(GeoUtils.capitalizeGps(nullLocationProvider));
+        assertNull(capitalizeGps(nullLocationProvider));
+    }
+
+    @Test
+    //Cm accuracy #4198
+    public void locationAccuracyIsFormattedInAppropriateUnit() {
+        final Context context = ApplicationProvider.getApplicationContext();
+        for (double accuracy : TEST_ACCURACIES) {
```

Comment:
I think instead of using a "table test" style here it might be simpler to have separate tests for metres and centimetres cases (`getAccuracyUnitString_returnsAccuracyInMeters` and `getAccuracyUnitString_whenAccuracyBelowOne_returnsAccuracyInCentimetres`). I think you only need to test one value for each unless I'm missing some edge cases.

Also, as a general rule we'd want to keep the test fixtures (`TEST_ACCURACIES` here) in test code rather than our actual app code. I think I'd prefer something like that to just live in the test class it's used in.

Reply:
Same goes for `GeoWidgetUtilsTest`